l10n_it_fatturapa_in_rc/wizard/wizard_import_fatturapa.py

```diff
@@ -7,7 +7,9 @@ class WizardImportFatturapa(models.TransientModel):
     def _prepare_generic_line_data(self, line):
         retLine = {}
         account_tax_model = self.env["account.tax"]
-        if float(line.AliquotaIVA) == 0.0 and line.Natura.startswith("N6"):
+        if float(line.AliquotaIVA) == 0.0 and (
+            line.Natura.startswith("N6") or line.Natura == "N2.1"
```

Comment:
Come mai l'aggiunta del controllo `N2.1`?

Se è una nuova fix/funzionalità, la puoi inserire in un commit separato?
Nel caso sarebbe anche utile una issue di tracciamento perché è probabile che vada riportata anche nelle altre versioni supportate.

Reply:
La separo sicuramente, anche perché N2.1 sono le fatture RC verso stati EU, emesse.   In fase di ricezione, non so se sia possibile come caso ricevere dallo SdI una fattura con N2.1.  So solo che da quando sono passati dal singolo codice N6 a N6.x, alcuni casi di RC sono diventati N2.1.

Diciamo che se non succede mai, andrebbe rimosso, ma non crea problemi.